helix-term/src/commands.rs

```diff
@@ -2316,6 +2321,12 @@ fn collapse_selection(cx: &mut Context) {
         Range::new(pos, pos)
     });
     doc.set_selection(view.id, selection);
+
+    // clear highlights on search matches
+    // but set search_matches.old to "" so that if 'n' is pressed
+    // to continue to traverse matches, highlights resume
+    view.search_matches.clear();
+    view.search_matches.old = Regex::new("").unwrap();
```

Comment:
again... is `unwrap` safe to use here?

Reply:
As it is initialize code with empty string. I don't think unwrap will panic here in runtime. It's ok imo.